### PR TITLE
Add Scala Steward

### PR DESCRIPTION
**Pull request by mdedetrich**
Self explanatory

### CODE REVIEW

**Review by gmethvin — APPROVED** [review]

File: .github/workflows/scala-steward.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Launch Scala Steward
+
+jobs:
+  scala-steward:
+    runs-on: ubuntu-22.04
+    name: Launch Scala Steward
+    if: github.repository == 'apache/pekko-connectors-kafka'
+    steps:
+      - name: Launch Scala Steward
+        uses: scala-steward-org/scala-steward-action@v2
```
Comment (He-Pin) [inline .github/workflows/scala-steward.yml:15]: need the env too

Comment (mdedetrich) [reply .github/workflows/scala-steward.yml:15]: This repo doesn't need it, there is no `.jvm-opts` file or any equivalent

Comment (He-Pin) [reply .github/workflows/scala-steward.yml:15]: so the heap memory is enough now?

Comment (mdedetrich) [reply .github/workflows/scala-steward.yml:15]: Yes, its a very small project

**Pull request merged**